app/models/facility.rb

```diff
@@ -95,8 +95,7 @@ def dashboard_analytics(period: :month, prev_periods: 3, include_current_period:
     results = [
       query.registered_patients_by_period,
       query.total_registered_patients,
-      query.follow_up_patients_by_period,
-      query.total_calls_made_by_period
```

Comment:
Can the `total_calls_made_by_period` method and associate specs be removed now?

Reply:
The column is still being displayed in the whatsapp graphics. This method is being called there.

Reply:
Should it be removed from the whatsapp graphics as well? Let's check with T/D.